docs/reference.html

```diff
@@ -924,8 +924,8 @@ <h3 id='map-method'>Methods</h3>
 
 </section><section class='collapsable'>
 
-<h4 id='map-methods-for-layers-and-controls'>Methods for Layers and Controls</h4>
-
+<h4>Methods for Layers and Controls</h4>
+<div class="ghost-pointer" id='map-methods-for-layers-and-controls'></div>
```

Comment:
There are some implications with "ghost pointers":

- I think search engines use the names of elements with an ID in their search results:

  ![docs](https://user-images.githubusercontent.com/26493779/152682052-b79bc981-ac66-44a7-b4b5-17d7854b80ee.png)

  Will this change affect the search results in search engines?

- If an empty `<div>` is focused nothing will be announced to screen reader users. And yes, blind users are also developers. The creator of the [Audiom](https://audiom.xrnavigation.io/) <q>cross-sensory map component</q> which is based on Leaflet was created by a blind user.

Reply:
Thank you very much for your reply.

Question 1: The principle of Soso engine is to index the keywords of the page and then search, I think this change will not affect the normal search.
Question 2: I think you are right, maybe we can add the same content in the div and hide it, what do you think?

Reply:
Actually, the docs may have accessibility problems already, I haven't looked into it much yet. But we shouldn't introduce new problems.

> maybe we can add the same content in the div and hide it

Is it necessary to duplicate the content? Shouldn't CSS changes be sufficient to fix the issue? Perhaps modify the margins, scroll-margin, scroll-padding or other related properties?

Reply:
I've checked the information and haven't found a better way to solve this problem yet, unless we drop position:sticky.

Reply:
Positioning the document accurately is probably more important to me than position:sticky.

Reply:
Is there a better way to solve this problem from your side?

Reply:
So there seem to be 2 problems with the sticky headings:
- Clicking a link in the toc does not always scroll to the heading / top of that section
- Even when the heading is scrolled to, there's a too large gap due to:
  https://github.com/Leaflet/Leaflet/blob/48a38c797b19cefb28002f7d09199d947260106a/docs/docs/css/main.css#L13

/cc @avioli who introduced sticky headings in https://github.com/Leaflet/Leaflet/pull/7703.

Reply:
Current information indicates that position:sticky will cause the page to not find the ID correctly.

Reply:
So far it seems that only the first problem is encountered on my side for the time being.

Reply:
`reference.html` is generated from `npm run docs`, so if I understand it correctly this file shouldn't be modified directly in most cases. But someone else can confirm that.

Reply:
Yes, I think you are right, I'll think about it and see if there is a better way to solve this problem I'm having.

Reply:
I added a comment to the main thread of the PR